File: db/csharp/SalesManagement.Domain/Models/Sales.cs
namespace SalesManagement.Domain.Models
{
    /// <summary>
    /// 売上データ
    /// </summary>
    public class Sales
    {
        public string SalesNo { get; set; } = string.Empty;
        public DateTime SalesDate { get; set; }
        public string OrderNo { get; set; } = string.Empty;
        public string DepartmentCode { get; set; } = string.Empty;
        public DateTime StartDate { get; set; }
        public string CompanyCode { get; set; } = string.Empty;
        public string EmployeeCode { get; set; } = string.Empty;
        public string WarehouseCode { get; set; } = string.Empty;
        public int SalesAmount { get; set; }
        public int ConsumptionTax { get; set; }
        public string? SlipComment { get; set; }
        public DateTime CreatedAt { get; set; }
        public string CreatedBy { get; set; } = string.Empty;
        public DateTime UpdatedAt { get; set; }
        public string UpdatedBy { get; set; } = string.Empty;
    }
}

File: db/fsharp/SalesManagement.Domain/Models/Sales.fs
namespace SalesManagement.Domain.Models

open System

/// <summary>
/// 売上データ
/// </summary>
[<CLIMutable>]
type Sales = {
    SalesSlipNo: string          // 売上伝票番号（PK）
    SalesDate: DateTime          // 売上日
    OrderNo: string option       // 受注番号（FK、追跡用）
    CustomerCode: string         // 得意先コード（FK）
    CustomerBranch: int          // 得意先枝番（FK）
    EmployeeCode: string         // 社員コード（FK）
    SalesAmount: int             // 売上金額合計
    ConsumptionTax: int          // 消費税合計
    SlipComment: string option   // 伝票備考
    DepartmentCode: string       // 部門コード（FK）
    CreatedAt: DateTime          // 作成日時
    CreatedBy: string            // 作成者名
    UpdatedAt: DateTime          // 更新日時
    UpdatedBy: string            // 更新者名
}
